layout/new_geisha.php

```diff
@@ -31,7 +31,9 @@ function getNewGeishaLayout(System $system, bool $enable_mobile_layout): Layout
 	<title>Shinobi Chronicles RPG</title>
 
 	<link rel='stylesheet' type='text/css' href='{$system->getCssFileLink("style/geisha/geisha.css")}' />
```

Comment:
That is the next step and something I would prefer in this PR, but is also really tedious!
At the moment I have removing `geisha.css` about ~half finished locally but I'm thinking we make that a new branch after merging this PR since its blocking the creation of new pages (Jutsu Page, Manual).